tests/expectations/test_expectation_arguments.py

```diff
@@ -507,7 +507,6 @@ def test_result_format_configured_no_set_default_override(
         "unexpected_count": 0,
         "unexpected_percent": 0.0,
         "partial_unexpected_list": [],
-        "partial_unexpected_index_list": None,
```

Comment:
here is an example of the value being outputted for a `SparkDFExecutionEngine`,  which doesn't support `unexpected_index`

Reply:
@anthonyburdi this is the change that I mentioned in Slack